index.md

```diff
@@ -1,87 +1,43 @@
+Anyone with an interested and enthusiastic about application security is welcome. All meetings are free and open. **You do not have to be an OWASP member**.
 
-Next Meeting/Event <!-- You should keep this section as it will populate your meetup events -->
----------------------
-{% include chapter_events.html group=page.meetup-group %}
+Referrals to this website or to individual meetings to colleagues or acquaintances are welcome.
 
-<!-- You should delete this comment
+## Upcoming Events
 
-Standard Chapter Page Template
-This is an example of a Project or Chapter page.
-Please change these items to indicate the actual information you wish to present. In addition to this information, the 'front-matter' above the text should be modified to reflect your actual information.  An explanation of each of the front-matter items is below:
+Check out our [Upcoming Events](https://www.meetup.com/de-DE/OWASP-Stuttgart/events/)
 
-{front matter for this file}
+## Information about Stuttgart Chapter Events
 
-```
-- layout: This is the layout used by project and chapter pages.  You should leave this value as col-sidebar
-- title: This is the title of your project or chapter page, usually the name.  For example, OWASP Zed Attack Proxy or OWASP Baltimore
-- tags: This is a space-delimited list of tags you associate with your project or chapter.  If you are using tabs, at least one of these tags should be unique in order to be used in the tabs files (an example tab is included in this repo) 
-- region: This is the region you are in according to our data
-```
+### When?
 
-{copy for this file (index.md)}
-Replace the text above the commented area with your information in the format below:
-```
-## Welcome
-Include some information here about your chapter
+We will be running quarterly chapter meetups with the option to have additional events such as hands-on workshops. 
 
-## Participation
-The Open Worldwide Application Security Project (OWASP) is a nonprofit foundation that works to improve the security of software. All of our projects ,tools, documents, forums, and chapters are free and open to anyone interested in improving application security. 
+The dates will be picked randomly to avoid recurring conflicts with other events (meetups, private obligations, holiday etc.). Typically, we chose a day from Tuesday to Thursday in the week, since our experience in organizing events is, that Mondays and Fridays have a very high no-show rate 😬
```

Comment:
```suggestion
The dates will be picked randomly to avoid recurring conflicts with other events (meetups, private obligations, holiday etc.). Typically, we choose a day from Tuesday to Thursday for our events, as our experience in organizing them has shown that Mondays and Fridays often have a very high no-show rate. 😬
```